Makefile

```
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: MIT
+
+.PHONY: test test-bats lint fmt
+
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)